src/client/components/Post.jsx

```diff
@@ -0,0 +1,51 @@
+import "../styling/post.css"
+
+const Post = () => {
+    return (
+        <main className="postPage">
+            <div className="post">
+                <p className="postUsername">Username</p>
+                <p className="postDateTime">28/02/2022 1:18pm</p>
+                <p className="postContent">
+                    Okay so some YouTubers have pointed the star icon in the Japanese titles for the games and how said icons always hint at the new game gimmick mechanic.
+
+                    And the only thing that comes to my mind involving stars are constellations which in turn makes me think of Saint Seiya/Knights of the Zodiac, which is a series about warriors donning armor based on constellations.
+
+                    So my theory is that certain Pokemon will be able to evolve or form change similar to Zamazenta and Zacian, wearing 'armor' made of some kind. OR it's a total form change, acquiring a new typing.
+
+                    But that's my theory. What are you guys' theories.
```

Comment:
How are you going to make sure this `Post` component can be re-used for other posts?

Reply:
There will only be 1 post per page. So it'll be a route like /posts/1 and the single post component will be used for the single post